aiida_zeopp/parsers/network.py

```diff
@@ -61,22 +61,16 @@ def parse(self, **kwargs):
 
             with self.retrieved.open(fname, 'rb') as handle:
                 if parser is None:
-
-                    # just add file, if no parser implemented
-                    parsed = SinglefileData(file=handle)
-                    self.out(link, parsed)
-
-                    # workaround: if block pocket file is empty, raise an error
-                    # (it indicates the calculation did not finish)
                     if link == 'block':
-                        content = handle.read()
-
-                        if not content.strip():
+                        if not parsed.content().strip():
                             self.logger.error(
                                 "Empty block file. This indicates the calculation of blocked pockets did not finish."
                             )
                             empty_block = True
 
+                    parsed = SinglefileData(file=handle)
```

Comment:
this would have to be moved to its previous position.
please also reinstate the comments

Reply:
I just updated the file based on above suggestions.

Reply:
Thanks! you checked that this still fixes the issue, right?

Tests seem broken currently because the pull from singularity hub fails.
Bit strange - I just tested the command [1] locally and it works fine... hm

[1] `singularity pull shub://ltalirz/singularity-recipe-zeopp:ansible`

Reply:
I just did and there was a minor issue which is fixed now. 
https://github.com/pzarabadip/aiida-zeopp/blob/37ea9704ef57adaadd2df5879925246d9ce8504c/aiida_zeopp/parsers/network.py#L72
It needed to be ```if not parsed.get_content().strip()```